appengine/taskqueue-push/README.md

```diff
@@ -0,0 +1,39 @@
+# A Java Task Queue example for Google App Engine
+
+This sample demonstrates how to use the [TaskQueue API][taskqueue-api] on [Google App
+Engine][ae-docs].
+
+[taskqueue-api]: https://cloud.google.com/appengine/docs/java/javadoc/com/google/appengine/api/taskqueue/package-summary
+[ae-docs]: https://cloud.google.com/appengine/docs/java/
+
+## Running locally 
+This example uses the
+[Maven gcloud plugin](https://cloud.google.com/appengine/docs/java/managed-vms/maven).
+To run this sample locally:
+
+    $ mvn appengine:devserver
+
+Go to the site `localhost:8080` to add elements to the queue.  They will appear in the log as the result of the Enqueue servlet transmitting the data to the Worker servlet.
+
+## Deploying
+In the following command, replace YOUR-PROJECT-ID with your
+[Google Cloud Project ID](https://support.google.com/cloud/answer/6158840).
+
+    $ mvn appengine:update -Dappengine.appId=YOUR-PROJECT-ID -Dappengine.version=SOME-VERSION
+
+## Setup
```

Comment:
I really don't think so.

Okay, @lesv, now that I see the whole file, I'm just echoing the form @tswast uses.

- Running locally (obvious)
- Deploying (obvious, but this is the form used where the user specifies the project id and version on the command line
- Setup, which is deploying, but putting the parameters in a config uses.